tests/common/fixtures/elasticsearch.py

```diff
@@ -15,14 +16,20 @@
 def es_client():
     client = _es_client()
     yield client
+    # Propogate changes to all shards to make sure all annotations that were added get removed.
+    elasticsearch_dsl.Index(client.index, using=client.conn).refresh()
     client.conn.delete_by_query(
         index=client.index,
         body={"query": {"match_all": {}}},
         # This query occassionally fails with a version conflict.
         # Forcing the deletion resolves the issue, but the exact
         # cause of the version conflict has not been found yet.
         conflicts="proceed",
+        # Add refresh to propogate the mark for deletion to all shards.
+        refresh=True,
     )
+    # Assert the index is now empty.
+    assert elasticsearch_dsl.Search(using=client.conn, index=client.index).execute()['hits']['total'] == 0
```

Comment:
I presume you put this in for testing? I think we can trust the Elasticsearch APIs to do their job and make the tests a bit faster by removing this. If we did want to put a check in for whatever reason, it would probably be faster to use the `count` function rather than search, since we only care about the number of documents in the index and don't need to perform an actual search and get document IDs.